main/emacs/APKBUILD

```diff
@@ -0,0 +1,186 @@
+# Maintainer: stef <l0ls0fo2i@ctrlc.hu>
+# Contributor: Timo Teräs <timo.teras@iki.fi>
+
+pkgname=emacs
+pkgver=25.1
+pkgrel=0
+pkgdesc="The extensible, customizable, self-documenting real-time display editor"
+arch="noarch"
+url="http://www.gnu.org/software/emacs/emacs.html"
+license="GPL3"
+depends="hicolor-icon-theme desktop-file-utils"
+makedepends="autoconf automake linux-headers paxmark
+	librsvg-dev giflib-dev libxpm-dev gtk+2.0-dev gconf-dev alsa-lib-dev
+	imagemagick-dev glib-dev fontconfig-dev libpng-dev
+	libxml2-dev pango-dev tiff-dev libjpeg-turbo-dev ncurses-dev
+	ncurses-libs gnutls-dev libxaw-dev"
+subpackages="$pkgname-doc $pkgname-nox $pkgname-x11 $pkgname-gtk2"
+source="ftp://ftp.gnu.org/gnu/emacs/emacs-$pkgver.tar.xz
+        gamedir.patch
+        noaslr-dump.patch
+        musl.patch"
+
+_builddir="$srcdir/emacs-$pkgver"
```

Comment:
Please rename it to `builddir`. It has been changed quite recently, `builddir` is now an abuild-defined variable.